app/src/main/java/com/infomaniak/swisstransfer/ui/OnboardingActivity.kt

```diff
@@ -17,13 +17,51 @@
  */
 package com.infomaniak.swisstransfer.ui
 
+import android.content.Intent
+import android.graphics.Color
+import android.os.Build
 import android.os.Bundle
 import androidx.activity.ComponentActivity
+import androidx.activity.SystemBarStyle
+import androidx.activity.compose.setContent
 import androidx.activity.enableEdgeToEdge
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.rememberCoroutineScope
+import com.infomaniak.swisstransfer.ui.screen.onboarding.OnboardingScreen
+import com.infomaniak.swisstransfer.ui.theme.SwissTransferTheme
+import com.infomaniak.swisstransfer.ui.utils.AccountUtils
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.launch
+import javax.inject.Inject
 
+@AndroidEntryPoint
 class OnboardingActivity : ComponentActivity() {
+
+    @Inject
+    lateinit var accountUtils: AccountUtils
+
     override fun onCreate(savedInstanceState: Bundle?) {
-        enableEdgeToEdge()
         super.onCreate(savedInstanceState)
+        enableEdgeToEdge(navigationBarStyle = SystemBarStyle.auto(Color.TRANSPARENT, Color.TRANSPARENT))
```

Comment:
Shouldn't this line be before the `super.onCreate()` ?

kDoc :
<img width="698" alt="image" src="https://github.com/user-attachments/assets/269780c3-3364-41b1-9894-1d903e4b14ff">

Reply:
What matters is only calling it before `setContent`. `super.onCreate` doesn't actually matter. We can indeed call it before, but like most other lines of code. You can also observe that when creating a new empty project in Android Studio the line is also placed after `super.onCreate`. 

If not important, I'd rather have it called after `super.onCreate` because, when you see a line of code before `super.onCreate` it makes you believe that placing it here was absolutely required, which is not the case here.

<img width="846" alt="Capture d’écran 2024-12-03 à 13 07 11" src="https://github.com/user-attachments/assets/5485733e-26f2-44ac-9a4d-9831be0bfcb8">
https://developer.android.com/develop/ui/views/layout/edge-to-edge#enable-edge-to-edge-display